{"name": "test_alert2_failure", "status": "passed", "steps": [{"name": "Нажатие кнопки для вызова Alert 2", "status": "passed", "start": 1760096492615, "stop": 1760096492726}, {"name": "Отменить Cancel у Alert 2", "status": "passed", "start": 1760096492728, "stop": 1760096492732}, {"name": "Подтверждаем, что после нажатия Отмены у нас выпало нужное предложение", "status": "passed", "steps": [{"name": "Получить текст после нажатия Alert 2", "status": "passed", "start": 1760096492766, "stop": 1760096492785}], "start": 1760096492733, "stop": 1760096492785}], "start": 1760096492574, "stop": 1760096492786, "uuid": "9a6a66c8-694b-4b01-ba77-9765d299fbdb", "historyId": "ff70814ac359c54dad1faef5f68cde8f", "testCaseId": "ff70814ac359c54dad1faef5f68cde8f", "fullName": "test_alerts.TestAlertsPage#test_alert2_failure", "labels": [{"name": "feature", "value": "Alerts"}, {"name": "story", "value": "Тест второго Alert (Негативный - нажатие отмены)"}, {"name": "suite", "value": "test_alerts"}, {"name": "subSuite", "value": "TestAlertsPage"}, {"name": "host", "value": "DESKTOP-T28TT8C"}, {"name": "thread", "value": "11056-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_alerts"}], "titlePath": ["test_alerts.py", "TestAlertsPage"]}